src/pages/saksbehandling/lukkSøknad/LukkSøknad.tsx

```diff
@@ -209,4 +209,17 @@ function hentOpprettetDatoFraSøknad(søknad: Søknad) {
     return søknad.opprettet;
 }
 
-export default LukkSøknad;
+const lukkSøknadBegrunnelseI18nId = (type: LukkSøknadOgAvsluttSøknadsbehandlingType): keyof typeof nb => {
```

Comment:
Jeg har begynt å legge inn slike keys direkte i ordlista, slik at man slipper den mapping-metoden her, i tillegg til at man for typesikring at alle keys har en tekst i ordlista. typ
```
//messages.ts
const messages: { [key in LukkSøknadBegrunnelse]: string } = {
    Trukket: 'Trukket',
    Bortfalt: 'Bortfalt',
    ...
};

export default messages;
```
Da kan man bruke de enkelt med
`formatMessage(LukkSøknadBegrunnelse.Trukket)`

Det synes hvert fall jeg at er nice, så slipper man det mellomsteget med denne metoden 😄

Reply:
Denne måten å gjøre ting på, er ikke alltid det som først kommer til tanken min, men ser veldig use caset for den ja!

I dette tilfellet her, så gjør vi en mapper vi over en enum, og blir da vanskelig å unngå det mellomsteget.